docker/Dockerfile-centos-7

```diff
@@ -45,4 +45,10 @@ RUN Install-Module -Name Microsoft.AzureStack.ReadinessChecker -RequiredVersion
 # create AzureRmContextSettings.json before it was generated
 COPY ${CONFIG}/${AZURERM_CONTEXT_SETTINGS} ${AZURE}/${AZURERM_CONTEXT_SETTINGS}
```

Comment:
@bganapa do we need this AzureRMContextSetting for Az too?

Reply:
Az settings are stored in a file still called as AzureRM*

Reply:
@TheOnlyWei IIRC, we did this as a workaround, do we still need to copy?

Reply:
@bganapa I am not sure, I never don't remember using AzureRM or copying context files when I worked on the container. I used Az.